src/components/editor/property/InputLookup.jsx

```diff
@@ -46,7 +46,7 @@ const InputLookup = (props) => {
   const lookupSelection = props.lookupValues.map((lookupValue) => (
     <div key={lookupValue.key} className="lookup-value">
       <span key={lookupValue.key}>{lookupValue.label || lookupValue.literal}</span>
-      <a href={lookupValue.uri}>
+      <a href={lookupValue.uri} aria-label="Link to value URI" target="_new">
```

Comment:
I would make the aria-label `Link to ${lookupValue.uri` to be consistent with our other aria labels.